examples/harmbench/README.md

```diff
@@ -0,0 +1,42 @@
+# HarmBench
+
+HarmBench is a standardized evaluation framework designed by the Center for AI Safety for automated red teaming of Large Language Models (LLMs). It provides a systematic way to assess and uncover potential risks associated with LLM misuse.
+
+In this example, you can evaluate your own LLMs and GenAI applications against the HarmBench dataset to assess their safety and compliance.
+
+## Setup
+
+1. Set your API keys in your environment:
+
+```bash
+export OPENROUTER_API_KEY=your_key_here
+export OPENAI_API_KEY=your_key_here
+```
+
+2. Download the HarmBench behaviors dataset:
+
+```bash
+./fetch_behaviors.sh
+```
+
+> **Note**: The complete HarmBench dataset is quite large. Running evaluations against it may incur significant API costs. Consider testing with a smaller subset first.
```

Comment:
how?

Reply:
Updated the documents to reflect the sample set.